packages/instantsearch.js/src/lib/utils/createSendEventForHits.ts

```diff
@@ -186,7 +186,7 @@ export function createBindEventForHits({
       index,
       methodName: 'bindEvent',
       args,
-      isSearchStalled: false,
+      invalidStatus: false,
```

Comment:
Why is the value different in `bindEvent` and `sendEvent`?

Reply:
I believe that's a bug introduced in https://github.com/algolia/instantsearch/commit/1686dfb096cfce062e268feda7956e3b160bf2da

Reply:
actually, no it's not a bug. BindEvent always should get its info, it's sendEvent(view) which should be skipped if the status is not right. The logic is in here, but on further thought, it probably should be in sendEvent directly